lib/loader/ldmgr.cpp

```diff
@@ -14,12 +14,9 @@ Expect<void> LDMgr::setPath(const std::filesystem::path &FilePath) {
   }
 
   const auto IntrinsicsTable = getSymbol<const void *>("intrinsics");
-  if (unlikely(!IntrinsicsTable)) {
```

Comment:
I think we should keep this error checker.

Reply:
This is for the case of AOT-WASM without intrinsics table.